docs/content/management-bundles.md

```diff
@@ -252,6 +252,182 @@ and number keys are converted to strings. Also, YAML !!binary tags are not
 supported.
 {{< /info >}}
 
+### Bundle Container Format
+
+OPA is able to interact with [OCI](https://opencontainers.org/) compatible registries to be able to download and use policies stored as containers.
+To configure OPA to use an OCI repository see the [service configuration section](../configuration/#services)
+
+**Structure**
+The bundle container is composed of 3 layers:
+- the manifest layer - contains the information about the tarball layer of the container(the digest, size, mediatype and annotations) and the config layer
+- the bundle tarball layer - the actual bundle tarball
+- the configuration layer - currently empty
+
+For OCI compatible registries an ***oci*** folder is created in the [persistence directory](../configuration/#miscellaneous). If this value is not set, because the OCI downloader plugin requires a storage path, the system's temporary folder location will be used instead.
```

Comment:
Should the user care about this folder? Should it be backed-up or cleaned out regularly? 💭

Reply:
Added a comment that this folder should be maintained by the user and it should be backed-up/cleaned periodically as this also acts as a cache for the OCI downloader